Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -163,20 +198,42 @@ private let dateLabelStyle: LabelStyle = { (label: UILabel) in
     |> \.numberOfLines .~ 0
 }
 
-private let descriptionLabelStyle: LabelStyle = { (label: UILabel) in
-  label
-    |> \.text  %~ { _ in Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life() }
-    |> \.textColor .~ UIColor.ksr_text_dark_grey_500
-    |> \.font .~ UIFont.ksr_caption1()
+private let learnMoreTextViewStyle: TextViewStyle = { (textView: UITextView) -> UITextView in
+  _ = textView
+    |> \.attributedText .~ attributedLearnMoreText()
+    |> \.isScrollEnabled .~ false
+    |> \.isEditable .~ false
+    |> \.isUserInteractionEnabled .~ true
     |> \.adjustsFontForContentSizeCategory .~ true
-    |> \.numberOfLines .~ 0
+
+  _ = textView
+    |> \.textContainerInset .~ UIEdgeInsets.zero
+    |> \.textContainer.lineFragmentPadding .~ 0
+    |> \.linkTextAttributes .~ [
+      .foregroundColor: UIColor.ksr_green_500
+  ]
+
+  return textView
 }
 
-private let learnMoreLabelStyle: LabelStyle = { (label: UILabel) in
-  label
-    |> \.text  %~ { _ in "\(Strings.Learn_more_about_accountability())." }
-    |> \.textColor .~ UIColor.ksr_green_500
-    |> \.font .~ UIFont.ksr_caption1()
-    |> \.adjustsFontForContentSizeCategory .~ true
-    |> \.numberOfLines .~ 0
+private func attributedLearnMoreText() -> NSAttributedString {
+  let string = """
+  \(Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life())
+  \(Strings.Learn_more_about_accountability())
+  """ as NSString
+
+  let linkRange = string.range(of: Strings.Learn_more_about_accountability())
+  let stringRange = string.range(of: string as String)
+
+  let attributedString = NSMutableAttributedString(string: string as String)
+
+  let url = urlForHelpType(
+    HelpType.trust, baseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl
+  )
+
+  attributedString.addAttribute(.font, value: UIFont.ksr_caption1(size: 12), range: stringRange)
```

Comment:
I think the default font size for `.ksr_caption1` is 12pts so you don't need to explicitly pass that in here.